canonical-web-and-design/ubuntu.com
# Restyle site search to match the design

## Done
Restyle site search to match [the design](https://app.zeplin.io/project/5ffdb21a3714151de9cf750c/screen/5ffdb2ce647d0519ad789d5d).

## QA
- Open the demo
- Click on the search and see it reveals the search box and focuses
- When you type a close icon appears which resets the search
- Check on small screens.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9290
